app/src/lang/translations/en-US.yaml

```
@@ -2372,3 +2372,49 @@ export_dashboard: Export Dashboard
 duplicate_dashboard: Duplicate Dashboard
 search_dashboard: Search Dashboard...
 search_flow: Search Flow...
+onboarding:
+  privacy_note:
+    We take user privacy very seriously. We'll never sell or share your information with anyone outside of our
+    organization.
+  action:
+    skip: Skip
+    first: Start Setup
+    continue: Continue
+    saveAndContinue: Save and Continue
+  welcome:
+    title: Welcome to Directus!
+    text: |
+      Whether you're starting from scratch, a seasoned pro, or just checking out Directus for the first time, we're so thrilled you're here. 
+
+      To get your project up and running as quickly as possible, the next few pages will guide you through initial user and project set-up. Ready? Let's get started!
+
+      By the way – Be sure to join our [Directus Discord](https://directus.chat) to get updates, community support, and to see what awesome stuff other users are building.
```

IMO, grammatically more correct:
```suggestion
      By the way, be sure to join our [Directus Discord](https://directus.chat) to get updates, community support, and to see what awesome stuff other users are building.
```